### ISSUE TITLE
Integration fails at startup after upgrade to HA 2024.2.0 (v3.0.0 Beta 3)

### ISSUE THREAD

**Issue by ismarslomic** [msp1974/homeassistant-jlrincontrol#130]
Just upgraded HA to `v2024.2.0` while running jlrincontrol `v3.0.0 Beta 3` and setup fails with following error message:

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 442, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/jlrincontrol/__init__.py", line 123, in async_setup_entry
    coordinator = JLRIncontrolUpdateCoordinator(hass, config_entry)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/jlrincontrol/coordinator.py", line 216, in __init__
    self.update_interval = config_entry.options.get(
    ^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 225, in update_interval
    self._update_interval_seconds = value.total_seconds() if value else None
                                    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'int' object has no attribute 'total_seconds'
```

**Comment by ismarslomic** [comment]
Change in HA where introduced in this PR: https://github.com/home-assistant/core/pull/107999

**Comment by ismarslomic** [comment]
In HA v2024.2.0 the `DataUpdateCoordinator` got new getter and setter for property `update_interval`, which is conflicting with the local property with identical name in JLRIncontrolUpdateCoordinator because of type `int` instead of `timedelta`.

**Comment by ismarslomic** [comment]
Solved in dev. Tested and verified that it resolved the issue in v2024.2.0
